src/main.ts

```diff
@@ -461,8 +461,7 @@ ${fs.readFileSync(summary.replayScriptPath, { encoding: "utf-8" }).split(/\r?\n/
 
             try {
                 console.log("Extracting commit SHA for repro steps");
-                const commit = (await execAsync(summary.repoDir, `git rev-parse @`)).trim();
-                text += `<li>In dir <code>${summary.repo.name}</code>, run <code>git reset --hard ${commit}</code></li>\n`;
+                text += `<li>In dir <code>${summary.repo.name}</code>, run <code>git reset --hard ${summary.commit}</code></li>\n`;
```

Comment:
This will end up being a behavior change; this try/catch was for the exec but now it doesn't do anything here, but can do something in the place it was moved to. I would try catch over in the new place to preserve this behavior.

Reply:
You're correct. Fixed. Instead, I now have the try-catch where the commit  is extracted and also assigned a nullable value in case we don't get it.